### PR TITLE
ports cobbychem or at least most of it

### PR DESCRIPTION
**Pull request by SomeguyManperson**
# General Documentation

### Intent of your Pull Request

ports the majority of the cobbychem c2 chemicals as well as the plant spaceman trumpet and galaxy thistle and their important reagents (this will become more important later)
gates trekchems behind a new chemical currently called lemoline that has to be ordered through cargo, replacing any use of bicaridine and kelotane with libital and aiuri since they're roughly the same chemical outside making tricord

does not port:
- granitaibarulre because it's slow as hell and i dont like it
- penthrite because i don't like it
- helbital because i also don't like it
- multiver for being too confusing for new players and mega snowflake
- TG synthflesh because we already did something with that
- renaming perfluorodecalin to convermol because i dont like it
- ~~removing or nerfing current chems because i can't care enough to do that right now~~

~~replaces charcoal with multiver because it's basically the same thing but multiver does something
multiver diluted stuff is diluted with shitty omnizine called protozine that heals 0.2 points per unit instead of 0.5 and can't be used to synthesize much of anything~~ lol

### Why is this change good for the game?
it's not
# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
ports cobbychem cat2 stuff, metafactor and peptides, as well as galaxy thistle and spaceman's trumpet as well as their important reagents polypyr and silibinin detailed on the TG wiki (syriniver renamed to thializid and hercuri renamed to rhigoxane)
also ports protozine which is omnizine except it heals less, currently unobtainable outside maint pills or funny plants
does NOT port helbital, penthrite, synthflesh, multiver or convermol
### What should players be aware of when it comes to the changes your PR is implementing?
new chemicals
lenturi is made with bromine instead of chlorine (otherwise it has the same recipe as silver sulf)
basic trekchems (bicard kelo antitox) are now synthesized with a special reagent orderable as a crate from cargo. Same recipe, just add the new reagent per unit
### What general grouping does this PR fall under? 
chemistry

# Changelog

:cl:  lots of tg coders and ExcessiveUseOfCobblestone
rscadd: adds a bunch of new medicines from tg
tweak: replaces most uses of trekchems with their cobbychem equivalent, and gates trekchems behind a new chemical that has to be ordered from cargo
rscadd: adds spaceman trumpet and galaxythistle plants
imageadd: adds images for trumpet and galaxythistle
/:cl:

### CODE REVIEW

**Comment by UselessTheremin** [comment]
You madman

**Comment by alexkar598** [comment]
> does not port:
> removing or nerfing current chems because i can't care enough to do that right now

Then this PR is pointless, the exact same thing happened when stypic powder and silver sulf got added, people managed to argue hard enough to keep bicard and kelo in and now everyone uses that and noone uses stypic and sulf except to cheese in combat via heal beakers.

**Comment by SomeguyManperson** [comment]
lol

**Comment by alexkar598** [comment]
Okay you tell me, between using situation chems which dont always heal and you sometimes need to do weird stuff and

chems that will heal brute no matter what

Which would you use?

**Review by patpol4 — APPROVED** [review]
Map is fine, its just swapping a path

**Comment by patpol4** [comment]
You should remove trekchems though imo, it's just chem bloat otherwise.

**Comment by SomeguyManperson** [comment]
one step at a time

**Comment by alexkar598** [comment]
"one step at a time"

Last words after goofchems got added and trekchems never got removed.

**Comment by SomeguyManperson** [comment]
people tend to prefer half measures over actually going all in and changing something in one pr. If I straight up removed trekchems right now I'd be getting bitched at by several more people probably saying somethign along the lines of "this makes chem massively less powerful" and/or "nooooooo dont remove the easy mode chems nooooooo" like with other stuff in the past so it's easier to simply add new stuff then change or remove the old stuff after a few weeks of having both so people get used to it rather than reading half of a pr and saying "change bad" because cobbychem does not have a very good history or reputation here

**Comment by ChesterTheCheesy** [comment]
Guys what if we replaced chem dispensers with cows that dispense reagents based on what you feed them?

**Comment by Moja10** [comment]
I think Cobbychem and Trekchem have a place together but it will take a lot of changes, adjustments, and nerfs so that chems aren't superior to other chems in function, side effects, healing ability, and recipe difficulty, but I would rather see a middle ground of both Cobby and Trek with adjustments than Cobby taking over everything. 

Switching to only Cobby will force us to pull in a bunch of /tg/ changes that aren't ideal, and force us to live with the adjustments they made for their community rather than considering how things can be different on Yogs.

**Comment by SomeguyManperson** [comment]
> I think Cobbychem and Trekchem have a place together but it will take a lot of changes, adjustments, and nerfs so that chems aren't superior to other chems in function, side effects, healing ability, and recipe difficulty, but I would rather see a middle ground of both Cobby and Trek with adjustments than Cobby taking over everything.
> 
> Switching to only Cobby will force us to pull in a bunch of /tg/ changes that aren't ideal, and force us to live with the adjustments they made for their community rather than considering how things can be different on Yogs.

that's the plan, going to add current cobbychems now then change stuff as the need arises

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm
Comment (redmoogle) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm:452]: How do you get this, dont see it made anywhere

Comment (SomeguyManperson) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm:452]: it was added here to be used in diluted syringes but I decided to not add the reagent used in those so it's maint pill fodder currently

**Comment by Yogbot-13** [comment]
#### Test Merge Deployed

##### Server Instance
main

##### Revision
Origin: fff1770e8b544be509edef7c14900647e0afebc6
Pull Request: 7f1756d807f6b3efb9cbdcba6076434a39ab5df1
Server: 47528dc0e18249af8fea62ba7c0d7339a00763f3

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: fff1770e8b544be509edef7c14900647e0afebc6
Pull Request: dce832201160cae38385a6280fa2893a12329189
Server: 02583086b1247c9a1be51165afc516aad0e5e7a7

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 17d3b574ef653d3278c26a59dc8418bee32d4849
Pull Request: dce832201160cae38385a6280fa2893a12329189
Server: 3dc4353d50b708e98f7a70c983f21bd3009239e1

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: b282edb69de8b2d8900b7573ebac641561d392d3
Pull Request: dce832201160cae38385a6280fa2893a12329189
Server: 461b198c2ad376e0a8dcba584d5cabc8cd93b91d

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: f1ccd63a36b575f6b21d51678f597a016dea4341
Pull Request: dce832201160cae38385a6280fa2893a12329189
Server: 9ae5169f85f4615a97cfce794663b0fcd3630291

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: f1ccd63a36b575f6b21d51678f597a016dea4341
Pull Request: dce832201160cae38385a6280fa2893a12329189
Server: ff51cbb8edef33f36a4b21e1679264ce72f404e8

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 77b18567de776b91f70821b0cb89efcea50e5638
Pull Request: dce832201160cae38385a6280fa2893a12329189
Server: 74d901727666ecdb4a83807850b62b7ae4596cc2

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: d9fa53dbf20373162cd911ee6c408579cbb08dd4
Pull Request: dce832201160cae38385a6280fa2893a12329189
Server: 71059923354c07cd59b391346f037e2cc0a88638

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: d9fa53dbf20373162cd911ee6c408579cbb08dd4
Pull Request: dce832201160cae38385a6280fa2893a12329189
Server: e241a029bff0a862598d4cf5330270ab7a61901c

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: ef0db3a25ba81dee1c39ea8fc8162a7a4a752ce0
Pull Request: dce832201160cae38385a6280fa2893a12329189
Server: aaf2a4de3a62a53112d314c3f9281454f06ce656

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 61be296c7775d9ea88e9b8a729456b211e4cc029
Pull Request: dce832201160cae38385a6280fa2893a12329189
Server: 7556b2e6eebbdd8cd977546ea7280d0bb63bd23f

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 61be296c7775d9ea88e9b8a729456b211e4cc029
Pull Request: dce832201160cae38385a6280fa2893a12329189
Server: 30cee7883419fedc2b14d05329f2362084d4a522

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 846b6d0591d4fdfa09eb9564a32ffc4691283565
Pull Request: dce832201160cae38385a6280fa2893a12329189
Server: a3380909465cd8b2082b69723d93038188c356e5

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 846b6d0591d4fdfa09eb9564a32ffc4691283565
Pull Request: dce832201160cae38385a6280fa2893a12329189
Server: 952eb07ef4c61276755bc1d018d27c948afb0e36

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 846b6d0591d4fdfa09eb9564a32ffc4691283565
Pull Request: dce832201160cae38385a6280fa2893a12329189
Server: 9a2ec9b6425c6d90690405f9c3b94f4666402c94

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: dd1d7fc33764ab1f7a35cf63a176bb8ea99c5bb7
Pull Request: dce832201160cae38385a6280fa2893a12329189
Server: 14cdaf78532bb345dde5558d0304548c1bc44f79

**Comment by Cartlord** [comment]
You failed to mention any real reason this was good for the game.

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: dd1d7fc33764ab1f7a35cf63a176bb8ea99c5bb7
Pull Request: dce832201160cae38385a6280fa2893a12329189
Server: 38094ac42f2645436ff266e4acf28dbe0a1101bc

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: a6e2415c469ba1af6e29b54aa239ed65b8c36ee7
Pull Request: dce832201160cae38385a6280fa2893a12329189
Server: d24b215e0e8c6c5423888ae49e4f29a9cf171b3c

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 66b7150aeab900c3f31c14b1bd1e42b0bf923c18
Pull Request: dce832201160cae38385a6280fa2893a12329189
Server: ef2e93fba57b1a0477b759c6d1a9a04f2aa0c565

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 158c00ddf79f3de3a5da5964b55ac17a89978863
Pull Request: dce832201160cae38385a6280fa2893a12329189
Server: b8c32c664435221e3bc7144827e3654e90e4e6dc

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: ca59602e0478d0927aad7a9b1fd4711bde1f9c82
Pull Request: dce832201160cae38385a6280fa2893a12329189
Server: c2377bfb63991a30acd46b8f81123ed6b04133b5

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 39c610f599c05ab55c9551c8328eb9cd5b838117
Pull Request: dce832201160cae38385a6280fa2893a12329189
Server: 576b980d7860d3a2d8af75956984af14a64580ef

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Revision
Origin: 39c610f599c05ab55c9551c8328eb9cd5b838117
Pull Request: dce832201160cae38385a6280fa2893a12329189
Server: ce13dee3d6e20b89dc7daedfb859ee58550b7dcf

**Review by redmoogle — APPROVED** [review]
Code looks good, there are silent merge conflicts so push something

**Review by nesisknesis — APPROVED** [review]

**Comment by JamieD1** [comment]
Will merge when Wiki is updated

**Review by JamieD1 — CHANGES_REQUESTED** [review]
Requires Wiki confirmation before merge

File: code/modules/reagents/chemistry/recipes/medicine.dm
Comment (alao3alao3) [inline code/modules/reagents/chemistry/recipes/medicine.dm]:
```suggestion
	results = list(/datum/reagent/medicine/bicaridine = 4)
```
```suggestion
	results = list(/datum/reagent/medicine/kelotane = 4)
```
```suggestion
	results = list(/datum/reagent/medicine/antitoxin = 4)
```
Needs to be changed now that it takes lemoline

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**
